app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/MainFragment.kt

```diff
@@ -21,7 +21,8 @@ class MainFragment : Fragment() {
 //    private lateinit var applicationViewModel: ApplicationViewModel
 
     override fun onCreateView(
-        inflater: LayoutInflater, container: ViewGroup?,
+        inflater: LayoutInflater,
```

Comment:
Reformatted to trailing comma to conform with all other function declarations in the project: https://kotlinlang.org/docs/functions.html#parameters